metricbeat/docs/fields.asciidoc

```diff
@@ -62308,6 +62308,75 @@ format: bytes
 
 --
 
+[float]
+=== io
+
+Disk I/O Metrics, as forwarded from /proc/[PID]/IO. Available on Linux only.
```

Comment:
I was looking on my fs and the whole path is lowercase (e.g: `/proc/2847/io`), I believe the docs should reflect it as well.
```suggestion
Disk I/O Metrics, as forwarded from /proc/[PID]/io. Available on Linux only.
```